transport/conn.go

```diff
@@ -454,6 +455,16 @@ func (c *Conn) sendRequest(req frame.Request, compress, tracing bool) (frame.Res
 	return resp.Response, resp.Err
 }
 
+func (c *Conn) useKeyspace(keyspace string) error {
+	s := &Statement{Content: `USE "` + keyspace + `"`}
+	_, err := c.Query(*s, nil)
+	if err != nil {
+		return err
+	}
+	c.keyspace = keyspace
```

Comment:
Unprotected access to a variable - aka race.
I think that technically there is no race here due to linearization in pool.

Let's remove the keyspace from Conn.